translations/messages.fr.xlf

```diff
@@ -0,0 +1,3099 @@
+        &lt;ol&gt;
+            &lt;li&gt;By as many squares as you want.&lt;/li&gt;
+            &lt;li&gt;Without going over another piece or stopping on another piece.&lt;/li&gt;
+            &lt;li&gt;Horizontally or vertically.&lt;/li&gt;
+            &lt;li&gt;Only the king can land on a throne.&lt;/li&gt;
+        &lt;/ol&gt;
+        To move a piece, click on it and then on its landing square.&lt;br/&gt;&lt;br/&gt;
+        This is the initial board, do the first move.</source>
+        <target>Au Tablut, toutes les pièces se déplacent de la même façon.
+        De façon équivalente aux tours aux échecs, une pièce se déplace :
+        &lt;ol&gt;
+            &lt;li&gt;D&apos;autant de case qu&apos;elle veut.&lt;/li&gt;
+            &lt;li&gt;Sans passer à travers ou s&apos;arrêter sur une autre pièce.&lt;/li&gt;
+            &lt;li&gt;Horizontalement ou verticalement.&lt;/li&gt;
+            &lt;li&gt;Seul le roi peut s&apos;arrêter sur un trône.&lt;/li&gt;
+        &lt;/ol&gt;
+        Pour déplacer une pièce, cliquez dessus, puis sur sa destination.&lt;br/&gt;&lt;br/&gt;
+        Ceci est le plateau initial, faites le premier mouvement.</target>
+      </trans-unit>
+      <trans-unit id="6012770625680782650" datatype="html">
+        <source>Capturing a soldier (1/2)</source>
+        <target>Capturer un simple soldat (1/2)</target>
+      </trans-unit>
+      <trans-unit id="677441772205421573" datatype="html">
+        <source>All pieces except the king, attackers and defenders, are soldiers.
+        To capture them, they have to be sandwiched between two of your pieces.
+        By getting too close, an attacker&apos;s soldier is in danger.&lt;br/&gt;&lt;br/&gt;
+        Capture it.</source>
+        <target>Toutes les pièces, attaquantes comme défenseuses, sont des soldats, à l&apos;exception du roi.
+        Pour les capturer, il faut en prendre une en sandwich entre deux de vos soldats.
+        En s&apos;approchant trop, un pion de l&apos;envahisseur s&apos;est mis en danger.&lt;br/&gt;&lt;br/&gt;
+        Capturez le.</target>
+      </trans-unit>
+      <trans-unit id="1504890408061490574" datatype="html">
+        <source>Congratulations, that will teach him a lesson!</source>
+        <target>Bravo, ça lui apprendra !</target>
+      </trans-unit>
+      <trans-unit id="3980320770441466216" datatype="html">
+        <source>Failed, you missed the opportunity to capture an opponent.</source>
+        <target>Raté, vous avez raté l&apos;occasion de capturer un adversaire</target>
+      </trans-unit>
+      <trans-unit id="4346619065189143436" datatype="html">
+        <source>Capturing a soldier (2/2)</source>
+        <target>Capturer un simple soldat (2/2)</target>
+      </trans-unit>
+      <trans-unit id="7441310005881510130" datatype="html">
+        <source>A second way to capture a soldier is against an empty throne.
+        The King has moved and endangered one of its soldiers..&lt;br/&gt;&lt;br/&gt;
```

Comment:
double dot is sms langage :D !

Reply:
King ou king tu dirais ici ?

Reply:
Je ne sais pas, y a plus d'occurence de king mais King ça fait classe.